src/ui/qgsoptionsbase.ui

```diff
@@ -925,6 +888,155 @@
+                  <item row="1" column="0">
+                   <widget class="QLabel" name="mTrustedProjectsFoldersLabel">
+                    <property name="text">
+                     <string>Trusted projects and folders allowed embedded code execution</string>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="1" column="1">
+                   <spacer>
+                    <property name="orientation">
+                     <enum>Qt::Horizontal</enum>
+                    </property>
+                    <property name="sizeHint" stdset="0">
+                     <size>
+                      <width>31</width>
+                      <height>20</height>
+                     </size>
+                    </property>
+                   </spacer>
+                  </item>
+                  <item row="1" column="2">
+                   <widget class="QToolButton" name="mBtnAddTrustedProject">
+                    <property name="toolTip">
+                     <string>Add new trusted project</string>
+                    </property>
+                    <property name="text">
+                     <string/>
+                    </property>
+                    <property name="icon">
+                     <iconset resource="../../images/images.qrc">
+                      <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="1" column="3">
+                   <widget class="QToolButton" name="mBtnRemoveTrustedProject">
+                    <property name="toolTip">
+                     <string>Remove trusted project or folder</string>
+                    </property>
+                    <property name="text">
+                     <string/>
+                    </property>
+                    <property name="icon">
+                     <iconset resource="../../images/images.qrc">
+                      <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="2" column="0" colspan="4">
+                   <widget class="QListWidget" name="mTrustedProjectsFoldersList">
+                    <property name="minimumSize">
+                     <size>
+                      <width>0</width>
+                      <height>90</height>
+                     </size>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="3" column="0">
+                   <widget class="QLabel" name="mUntrustedProjectsFoldersLabel">
+                    <property name="text">
+                     <string>Untrusted projects and folders denied embedded code execution</string>
```

Comment:
```suggestion
                     <string>Untrusted projects and folders denied embedded script execution</string>
```

Reply:
@nyalldawson would this work in this situation (also pairing with the above "allowing" tense)? 
`<string>Untrusted projects and folders denying embedded script execution</string>`